new/PRS13996KidanaM.xml

```diff
@@ -47,11 +47,30 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <person>
                     <persName xml:lang="gez" xml:id="n1">ኪዳነ፡ ማርያም፡</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1">Kidāna Māryām</persName>
+                    <persName type="alt" xml:id="n2">Kidāna Māriām</persName>
+                    <persName type="alt" xml:id="n3">Kidāna-Mārjām</persName>
+                    <floruit when="1868">Kidāna Māryām was an educated native of <placeName ref="LOC1264Adwa"/> active in 
+                        <placeName ref="LOC3957Jerusa"/>
+                        at the time of <persName ref="PRS7812Peterman"/>'s stay there as the Prussian vice consul in <date when="1868">1868</date>. 
+                        He became
+                    Petermann's informant, supplying him with material on the study of Tigrinya and Amharic and engaging in the copying and 
+                    acquisition of manuscripts, probably also after Petermann's return to Europe.</floruit>
```

Comment:
Perhaps it is useful to clarify, that Kidana Maryam provided (mostly or exclusively) Gəʿəz manuscripts to Petermann. In the current phrase, one can have the impression, that Petermann received Amharic and Tigrinya manuscripts from Kidana Maryam. You may write: "He became Petermann's informant, supplying him with material on the study of Tigrinya and Amharic and engaging in the copying and acquisition of Gəʿəz-manuscripts, probably also after Petermann's return to Europe." or you may turn it to: 
"He became Petermann's informant, who was engaged in copying and acquisition of manuscripts and also supplied him with material on the study of Tigrinya and Amharic, probably even after Petermann's return to Europe."

Reply:
Thank you, that is a good point, I updated the wording